pipeline.go

```diff
@@ -104,8 +104,28 @@ func generatePipeline(steps []interface{}, pipelineEnv map[string]string, projec
 
 		env, foundEnv := stepMap["env"].(map[interface{}]interface{})
 		_, foundBlockStep := stepMap["block"].(string)
+		_, foundTriggerStep := stepMap["trigger"].(string)
 
-		if !foundBlockStep {
+		if foundTriggerStep {
+			// Is there a build property
+			build, foundBuild := stepMap["build"].(map[interface{}]interface{})
+
+			if (!foundBuild) {
+				build = make(map[interface{}]interface{}) // TODO allow nesting
```

Comment:
Can you please clarify this TODO?
This will help other people pick this up.

Reply:
Apologies, I think it's errant. I'll remove.